ci/rpm/build.sh

```diff
@@ -24,16 +24,15 @@ SCONS_ARGS="${SCONS_FAULTS_ARGS}"
 
 : "${CHROOT_NAME:='centos+epel-7-x86_64'}"
 : "${TARGET:='centos7'}"
+: "${REPO_SPEC:='el7'}"
```

Comment:
Should this default be `el-7`?
```suggestion
: "${REPO_SPEC:='el-7'}"
```

Reply:
That is only used for debugging outside of Jenkins, but yes, it should be el-7.
Fixed in the current push.